src/templates/scannerExamples.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { graphql } from "gatsby";
+import Layout from "../components/Layout";
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import 'react-tabs/style/react-tabs.css';
+
+const ScannerExamples = props => {
+  const path = `${props.path.substring(13)}/`;
```

Comment:
Why 13?

Reply:
I was removing part of the string for display purposes. The rather unorthodox approach was not necessary though. Good catch.